src/content/changes/highlights/2023-11-16.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2023-11-16T14:00:00
+title: Highlights for the week ending Nov 16 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+## Apps
+
+### Falco
+
+[`Falco`](https://github.com/giantswarm/falco-app) [`0.7.0`](https://github.com/giantswarm/falco-app/releases/tag/v0.7.0) is released. Which means we now use versions newer than `0.36.0`. This update brings in the [`Falcoctl`](https://github.com/falcosecurity/falcoctl) tool. From now on `Falco` Images are no longer shipped with rules inside the image, instead they will now use an init container to download the rules from an rules repository (by default, this is the one from Falco themselves) and will check every six hours whether updates have taken place.
+The amount of rules `Falco` installs by default has drastically been lowered, and the previous ruleset has been devided in the categories, `Standard`, `Incubating`, and `Sandbox`. `Falco` decided on this approach after community input showed an interest in a starter/default ruleset that was less noisy. 
+If the whole previous ruleset is still required, it's possible to enable this in the `Falcoctl` configuration in the helm chart. For more information about the new situation regarding rules, check Falco's new [`rules repository`](https://github.com/falcosecurity/rules). 
```

Comment:
```suggestion
[`Falco`](https://github.com/giantswarm/falco-app) [`0.7.0`](https://github.com/giantswarm/falco-app/releases/tag/v0.7.0) is released. It means the underlying component version is higher than `0.36.0`. The update contains [`falcoctl`](https://github.com/falcosecurity/falcoctl) tool which helps to administrate Falco configuration and audit the state of the system. From now on `Falco` images will not be longer shipped with rules inside the image. Instead, they will use an init container to download the rules from an official repository and will check frequently for updates. As a consequence, the amount of rules `Falco` installs has drastically been lowered, and the previous ruleset has been divided into several categories, `Standard`, `Incubating`, and `Sandbox`. This reduces the noise in general but in case the previous ruleset is required, it is possible to enable this using the command line tool. For more information about the new situation, check Falco's new [`rules repository`](https://github.com/falcosecurity/rules). 
```
@Strigix WDYT?